content/docs/reference/pkg/nodejs/pulumi/awsx/apigateway/_index.md

```diff
@@ -1575,95 +1580,39 @@ This value will be JSON.stringify'd as part of normal processing.  It should not
-
-The description of the REST API
-
-<h4 class="pdoc-member-header" id="RestApiArgs-endpointConfiguration">
-<a class="pdoc-child-name" href="https://github.com/pulumi/pulumi-awsx/blob/7e3319acc1119743eb7e923349542ed966380c18/nodejs/awsx/apigateway/api.ts#L431">property <b>endpointConfiguration</b></a>
-</h4>
-
-<pre class="highlight"><code><span class='kd'></span>endpointConfiguration?: <a href='/docs/reference/pkg/nodejs/pulumi/pulumi/#Input'>pulumi.Input</a>&lt;<a href='/docs/reference/pkg/nodejs/pulumi/aws/types/input/#RestApiEndpointConfiguration'>RestApiEndpointConfiguration</a>&gt;;</code></pre>
-
-Nested argument defining API endpoint configuration including endpoint type. Defined below.
-
-<h4 class="pdoc-member-header" id="RestApiArgs-minimumCompressionSize">
-<a class="pdoc-child-name" href="https://github.com/pulumi/pulumi-awsx/blob/7e3319acc1119743eb7e923349542ed966380c18/nodejs/awsx/apigateway/api.ts#L435">property <b>minimumCompressionSize</b></a>
-</h4>
-
-<pre class="highlight"><code><span class='kd'></span>minimumCompressionSize?: <a href='/docs/reference/pkg/nodejs/pulumi/pulumi/#Input'>pulumi.Input</a>&lt;<span class='kd'><a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number'>number</a></span>&gt;;</code></pre>
-
-Minimum response size to compress for the REST API. Integer between -1 and 10485760 (10MB). Setting a value greater than -1 will enable compression, -1 disables compression (default).
-
-<h4 class="pdoc-member-header" id="RestApiArgs-name">
-<a class="pdoc-child-name" href="https://github.com/pulumi/pulumi-awsx/blob/7e3319acc1119743eb7e923349542ed966380c18/nodejs/awsx/apigateway/api.ts#L413">property <b>name</b></a>
-</h4>
-
-<pre class="highlight"><code><span class='kd'></span>name?: <a href='/docs/reference/pkg/nodejs/pulumi/pulumi/#Input'>pulumi.Input</a>&lt;<span class='kd'><a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String'>string</a></span>&gt;;</code></pre>
-
-The name of the REST API.  Defaults to the name of the awsx.apigateway.Api if unspecified.
-
-<h4 class="pdoc-member-header" id="RestApiArgs-policy">
-<a class="pdoc-child-name" href="https://github.com/pulumi/pulumi-awsx/blob/7e3319acc1119743eb7e923349542ed966380c18/nodejs/awsx/apigateway/api.ts#L439">property <b>policy</b></a>
-</h4>
-
-<pre class="highlight"><code><span class='kd'></span>policy?: <a href='/docs/reference/pkg/nodejs/pulumi/pulumi/#Input'>pulumi.Input</a>&lt;<span class='kd'><a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String'>string</a></span>&gt;;</code></pre>
-
-JSON formatted policy document that controls access to the API Gateway.
-
```

Comment:
It's worth noting that this all goes away because we now reference the type from `@pulumi/aws` directly in `@pulumi/awsx`:

```typescript
export type RestApiArgs = aws.apigateway.RestApiArgs;
```